tagilmo/VereyaPython/timestamped_video_frame.py

```diff
@@ -82,4 +79,7 @@ def __init__(self, message: TimestampedUnsignedCharVector,
         self.calibrationMatrix = np.reshape(np.asarray(loadedjson['projectionMatrix'], dtype=np.dtype(numpy.float32)), (4,4))
         jo_len = jo_len + 4
         received_img_bytes = message.data[jo_len:]
-        self.pixels = cv2.cvtColor(cv2.imdecode(np.asarray(bytearray(received_img_bytes), dtype="uint8"), cv2.IMREAD_COLOR), cv2.COLOR_BGR2RGB)
\ No newline at end of file
+        self._pixels = received_img_bytes
+
+    def pixels(self):
+        return cv2.cvtColor(cv2.imdecode(np.frombuffer(self._pixels, dtype="uint8"), cv2.IMREAD_COLOR), cv2.COLOR_BGR2RGB)
```

Comment:
Is it necessary to convert colors? If I just `imshow` the results from `pixels()` the image colors are wrong with this conversion. If I remove this conversion, then everything is ok. If the conversion is not needed on Mac, but is needed on other platforms, then we need to make this conversion platform-dependent. Could you please check this on different platforms?

Reply:
@Necr0x0Der  well, I've changed that code to be platform-dependent, though i have to say, that on virtual mac (sosumi) this cv2.cvtColor is needed and with current platform-dependent conversion it gives wrong images (bgr).

Reply:
@Necr0x0Der well, it seems that we don't need this cvtcolor after all. I've re-checked it once more. On sosumi too. So i've just deleted this conversion.